prometheus/base/alerting/prometheus.argocd_alerts.yaml

```diff
@@ -17,3 +17,14 @@ spec:
       annotations:
         summary: "Application {{ $labels.name }} in namespace {{ $labels.dest_namespace }} was 'Degraded'"
         description: "Application {{ $labels.name }} in namespace {{ $labels.dest_namespace }} was 'Degraded' in the last 5 minutes"
+    - alert: ProgressingArgocdApp
+      expr: |
+       (avg_over_time(argocd_app_info{health_status="Progressing", dest_namespace!~".+-tenant"}[20m]) > 0.5)
+        and on(dest_namespace, name)
+        max_over_time(argocd_app_info{health_status!="Healthy", dest_namespace!~".+-tenant"}[4m]) == 1
+      for: 0m
+      labels:
+        severity: Critical
+      annotations:
+        summary: "Application {{ $labels.name }} in namespace {{ $labels.dest_namespace }} is in 'Progressing' state for more than 20 minuts"
```

Comment:
These are wrong summary and description, the application was in progress state for more than 50% of the previous 20 minutes.

Reply:
it changed, the rule has also changed so it's according to the new rule